how-it-works/2_chainkeytech/04_chain_key_tokens.subpage.md

```diff
@@ -0,0 +1,66 @@
+
+<figure>
+<img src="/img/how-it-works/chain_key_token_arch.png" alt="Architecture for chain-key tokens" title="Architecture for chain-key tokens" align="center" style="width:1000px">
+<figcaption>High-level architecture for chain-key tokens on ICP</figcaption>
+</figure>
+
+Note that the token ledger and minter functionalities can also be integrated into a single smart contract — a question of software architecture. However, splitting the functionality into two canisters as proposed is a preferable approach for the reason of modularity as the same ledger canister code can be reused for any chain-key token and only the minter needs to be adapted to the mechanics of the token of the underlying chain, particularly to account for differences between UTXO- and account-based underlying chains, like Bitcoin and Ethereum, respectively.
+
+### Creating Chain-key Tokens from Underlying Tokens
+
+When a user wants to obtain chain-key tokens for some underlying tokens, e.g., bitcoin, they send the underlying tokens to an address on the origin blockchain owned by the chain-key token smart contract (minter), instead of an intermediary as done for wrapping. The chain-key token smart contract has an address on the origin blockchain, which is made possible through chain-key cryptography, hence the name of chain-key tokens, or chain-key Bitcoin as a concrete example: In the Bitcoin example, using chain-key ECDSA signing technology — concretely, an advanced form of threshold ECDSA, the chain-key Bitcoin smart contract can have and use ECDSA key pairs, much like any user of the Bitcoin network, but fully on chain. That means the smart contract can obtain ECDSA public keys and from the public keys it derives addresses on the Bitcoin network, to which bitcoin can be sent on the Bitcoin network by anyone. Once this chain-key token smart contract has received Bitcoin from a user and the transaction has received a sufficient number of confirmations on the Bitcoin network, the canister instructs the token ledger for the chain-key token to create, or mint, an amount of chain-key Bitcoin tokens corresponding to the received amount of bitcoin. This approach leads to the chain-key token being 1:1 backed with the underlying token, which is held in on-chain custody by the minter canister smart contract.
```

Comment:
I think we use the Bitcoin example too many times. It suffices to state once (or twice) that Bitcoin is an instructive example.

Reply:
The whole section feels repetitive. Hasn't everything been said before? Maybe some parts above can be shortened to add more meat to this section?

Reply:
Yes, there's some repetition. May address this in another iteration after initial publication.

Reply:
Yes, there's some repetition. May address this in another iteration after initial publication.